apps/token-manager/app/src/App.js

```diff
@@ -121,7 +121,7 @@ class App extends React.Component {
               <AppBar
                 title={
                   <Title>
-                    <TitleLabel>Token</TitleLabel>
+                    <TitleLabel>Token Manager</TitleLabel>
```

Comment:
Mostly because of this discrepancy:

<img width="377" alt="screen shot 2018-12-14 at 2 54 53 pm" src="https://user-images.githubusercontent.com/4166642/50007144-457dcf00-ffb0-11e8-805e-e11c0f45ba45.png">

I'm also not sure when the Token Manager will be enhanced to support more than one token, and when we do so, I'm pretty sure the entire app will need to be rebuilt.